insets/src/main/java/dev/chrisbanes/accompanist/insets/Insets.kt

```diff
@@ -104,10 +104,89 @@ val AmbientWindowInsets = staticAmbientOf<WindowInsets> {
     error("AmbientInsets value not available. Are you using ProvideWindowInsets?")
 }
 
+/**
+ * This function sets up the necessary listeners on the given [view] to be able to observe
+ * [WindowInsetsCompat] instance dispatched by the system.
+ *
+ * This function is useful for when you prefer to handle the ownership of the [WindowInsets]
+ * yourself. One example of this is if you find yourself using [ProvideWindowInsets] in multiple
```

Comment:
I don't think that the "multiple" part is helpful here as the issue exhibits with a single fragment right? It's a timing issue.